 validate.c: In function 'print_range_list':
 validate.c:303: warning: 's' may be used uninitialized in this function

 validate.c |    6 ++++++
 1 files changed, 6 insertions(+)

Index: pdisk/validate.c
===================================================================
--- pdisk.orig/validate.c
+++ pdisk/validate.c
@@ -321,6 +321,9 @@ print_range_list(range_list *list)
 	    case kMultiplyAllocated:
 		s = "multiply allocated";
 		break;
+	    default:
+		s = "unknown";
+		break;
 	    }
 	    printed = 1;
 	    printf("\t%lu:%lu %s\n", cur->start, cur->end, s);
@@ -336,6 +339,9 @@ print_range_list(range_list *list)
 	    case kMultiplyAllocated:
 		s = "multiply allocated";
 		break;
+	    default:
+		s = "unknown";
+		break;
 	    }
 	    printed = 1;
 	    printf("\t%lu:%lu out of range, but %s\n", cur->start, cur->end, s);
